docs/zkapps/getting-started-zkapps.mdx

```diff
@@ -98,13 +99,47 @@ Add the logic for your smart contract.
     - If it works as expected with no errors, add more functionality.
     - If there are errors, look through the stack traces to find the source of the errors and update the contract to resolve them.
 
-### 5. Create an integration test.
+### 5. Create an integration test
 
 - Create a Node.js script to run the smart contract and test it's functionality, similar to step [3. Add testing code](#3-add-testing-code).
 
     For an example, see the Node.js script that runs the Tic Tac Toe smart contract in the [run.ts](https://github.com/o1-labs/zkapp-cli/blob/main/examples/tictactoe/ts/src/run.ts) file.
 
-### 6. Test with a live network
+- [Test zkApps locally](/zkapps/testing-zkapps-locally) with a simulated local blockchain.
+
+### 6. Test with Lightnet
+
+To test your zkApp with an accurate representation of Mina blockchainwith, start Lightnet:
+
+1. Spin up a Lightnet blockchain:
+
+    ```sh
+    zk lightnet start
+    ```
+
+    In most cases, a single node is appropriate for testing. Use the start configuration options appropriate for your use case. See [Start a local network with other settings](/zkapps/lightnet#start-a-local-network-with-other-settings).
+
+1. Verify the status of the local blockchain:
+
+    ```sh
+    zk lightnet status
+    ```
+
+1. Communicate with the Account Manager service to fetch account details.
```

Comment:
@shimkiv let's chat and walk through what this looks like for our dev

Reply:
This will be added before merging, right?